docs/topics/mta-web-reviewing-an-analysis-report.adoc

```diff
@@ -8,7 +8,7 @@
 
 An {ProductShortName} analysis report contains a variety of sections, including a listing of the technologies used by the application, the dependencies of the application, and the lines of code that must be changed to successfully migrate or modernize the application.
 
-For more information on the contents of an {ProductShortName} analysis report, see the link:{ProductDocUserGuideURL}#review-reports_cli-guide[Reviewing the reports]
+For more information on the contents of an {ProductShortName} analysis report, see the link:{ProductDocUserGuideURL}#review-reports_cli-guide[Reviewing the reports].
 
 .Procedure
```

Comment:
I believe that the snippet I created (https://github.com/windup/windup-documentation/pull/807) 
should go here to replace this procedure

Reply:
@emarcusRH - done. ta muchly